cmd/execution/main.go

```diff
@@ -330,6 +334,12 @@ func main() {
 				}
 			}
 
+			hwLogger := node.Logger.With().Str("system", "hardware").Logger()
+			err = logHardware(hwLogger)
```

Comment:
Wonder why do we log it inside of "execution state ledger"?

If it's logging static hardware spec, we could move it to its own module:

```
		Module("log hardware spec", func(node *cmd.NodeConfig) error {
			hwLogger := node.Logger.With().Str("system", "hardware").Logger()
			err = logHardware(hwLogger)
			if err != nil {
				hwLogger.Error().Err(err)
			}
			return nil
		}).
```

Reply:
Great idea!  Done in 22c46fcb90cc15bf508d23c58e9bc307db240de1.